eng/scripts/Language-Settings.ps1

```diff
@@ -416,3 +417,14 @@ function GetExistingPackageVersions ($PackageName, $GroupId=$null)
     return $null
   }
 }
+
+function Get-python-DocsMsMetadataForPackage($PackageInfo) { 
+  $docsReadmeName = Split-Path -Path $PackageInfo.DirectoryPath -Leaf
```

Comment:
`docsReadmeName` might need to have a leading `azure-` removed. See: https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Language-Settings.ps1#L266